core/src/main/java/org/apache/iceberg/ManifestGroup.java

```diff
@@ -180,43 +202,24 @@ ManifestGroup planWith(ExecutorService newExecutorService) {
         return CloseableIterable.transform(entries, e -> new BaseFileScanTask(
             e.file().copy(), deleteFiles.forEntry(e), schemaString, specString, residuals));
       }
-    });
-
-    if (executorService != null) {
-      return new ParallelIterable<>(tasks, executorService);
-    } else {
-      return CloseableIterable.concat(tasks);
-    }
+    };
   }
 
- /**
-   * Returns an iterable for manifest entries in the set of manifests.
-   * <p>
-   * Entries are not copied and it is the caller's responsibility to make defensive copies if
-   * adding these entries to a collection.
-   *
-   * @return a CloseableIterable of manifest entries.
-   */
   public CloseableIterable<ManifestEntry<DataFile>> entries() {
     return CloseableIterable.concat(entries((manifest, entries) -> entries));
   }
 
-  private <T> Iterable<CloseableIterable<T>> entries(
-      BiFunction<ManifestFile, CloseableIterable<ManifestEntry<DataFile>>, CloseableIterable<T>> entryFn) {
```

Comment:
@yohengyang, please revert all of the whitespace and non-functional changes. Otherwise, we can't really review this.